### PR TITLE
Update outdated https://en.pingcap.com/ links

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

**Note**: Another option would be to make en.pingcap.com a cname for www.pingcap.com in the DNS settings. I'm not sure which option is best.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by hfxsd — APPROVED** [review]

File: develop/dev-guide-proxysql-integration.md [hunk 1]
```diff
@@ -624,7 +624,7 @@ The following steps will set up ProxySQL and TiDB on ports `6033` and `4000` res
 
 ## Production environment
 
-For a production environment, it is recommended that you use [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) directly for a fully-managed experience.
+For a production environment, it is recommended that you use [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) directly for a fully-managed experience.
```
Comment (lilin90) [inline develop/dev-guide-proxysql-integration.md:627]: It jumps to https://www.pingcap.com/tidb-dedicated/. And the following steps are based on TiDB Dedicated.
```suggestion
For a production environment, it is recommended that you use [TiDB Dedicated](https://www.pingcap.com/tidb-dedicated/) directly for a fully-managed experience.
```

File: tidb-cloud/release-notes-2021.md [hunk 2]
```diff
@@ -91,7 +91,7 @@ Bug fixes:
 
 ## June 25, 2021
 
-* Fix the **Select Region** not working issue on the [TiDB Cloud Pricing](https://en.pingcap.com/products/tidbcloud/pricing/) page
+* Fix the **Select Region** not working issue on the [TiDB Cloud Pricing](https://www.pingcap.com/products/tidbcloud/pricing/) page
```
Comment (lilin90) [inline tidb-cloud/release-notes-2021.md:94]:
```suggestion
* Fix the **Select Region** not working issue on the [TiDB Cloud Pricing](https://www.pingcap.com/pricing/) page
```

File: tidb-cloud/release-notes-2022.md [hunk 3]
```diff
@@ -401,7 +401,7 @@ This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-c
 
 * Introduce [Credits](/tidb-cloud/tidb-cloud-billing.md#credits) (previously named as trail points) for PoC users.
 
-    You can now view information about your organization's credits on the **Credits** tab of the **Billing** page, the credits can be used to pay for TiDB Cloud fees. You can [contact us](https://en.pingcap.com/apply-for-poc/) to get credits.
+    You can now view information about your organization's credits on the **Credits** tab of the **Billing** page, the credits can be used to pay for TiDB Cloud fees. You can [contact us](https://www.pingcap.com/apply-for-poc/) to get credits.
```
Comment (lilin90) [inline tidb-cloud/release-notes-2022.md:404]: It jumps to https://www.pingcap.com/demo/. But it seems that it doesn't make sense here. I'll change it to email, as used in https://docs.pingcap.com/tidbcloud/tidb-cloud-poc.
```suggestion
    You can now view information about your organization's credits on the **Credits** tab of the **Billing** page, the credits can be used to pay for TiDB Cloud fees. You can <a href="mailto:tidbcloud-support@pingcap.com">contact us</a> to get credits.
```

File: tidb-cloud/release-notes-2022.md [hunk 4]
```diff
@@ -517,7 +517,7 @@ This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-c
 ## June 7, 2022
 
 * Add the [Try Free](https://tidbcloud.com/free-trial) registration page to quickly sign up for TiDB Cloud.
-* Remove the **Proof of Concept plan** option from the plan selection page. If you want to apply for a 14-day PoC trial for free, go to the [Apply for PoC](https://en.pingcap.com/apply-for-poc/) page. For more information, see [Perform a Proof of Concept (PoC) with TiDB Cloud](/tidb-cloud/tidb-cloud-poc.md).
+* Remove the **Proof of Concept plan** option from the plan selection page. If you want to apply for a 14-day PoC trial for free, go to the [Apply for PoC](https://www.pingcap.com/apply-for-poc/) page. For more information, see [Perform a Proof of Concept (PoC) with TiDB Cloud](/tidb-cloud/tidb-cloud-poc.md).
```
Comment (lilin90) [inline tidb-cloud/release-notes-2022.md:520]:
```suggestion
* Remove the **Proof of Concept plan** option from the plan selection page. If you want to apply for a 14-day PoC trial for free, <a href="mailto:tidbcloud-support@pingcap.com">contact us</a>. For more information, see [Perform a Proof of Concept (PoC) with TiDB Cloud](/tidb-cloud/tidb-cloud-poc.md).
```

File: tidb-cloud/release-notes-2022.md [hunk 5]
```diff
@@ -551,7 +551,7 @@ This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-c
 TiDB Cloud is now in General Availability. You can [sign up](https://tidbcloud.com/signup) and select one of the following options:
 
 * Get started with [Developer Tier](/tidb-cloud/select-cluster-tier.md#tidb-serverless) for free.
-* Apply for [a 14-day PoC trial for free](https://en.pingcap.com/apply-for-poc/).
+* Apply for [a 14-day PoC trial for free](https://www.pingcap.com/apply-for-poc/).
```
Comment (lilin90) [inline tidb-cloud/release-notes-2022.md:554]:
```suggestion
* <a href="mailto:tidbcloud-support@pingcap.com">Contact us</a> to apply for a 14-day PoC trial for free.
```

File: tidb-cloud/release-notes-2022.md [hunk 6]
```diff
@@ -572,7 +572,7 @@ General changes:
 * Introduce a new public region: `eu-central-1`.
 * Deprecate 8 vCPU TiFlash and provide 16 vCPU TiFlash.
 * Separate the price of CPU and storage (both have 30% public preview discount).
-* Update the [billing information](/tidb-cloud/tidb-cloud-billing.md) and the [price table](https://en.pingcap.com/tidb-cloud/#pricing).
+* Update the [billing information](/tidb-cloud/tidb-cloud-billing.md) and the [price table](https://www.pingcap.com/tidb-cloud/#pricing).
```
Comment (lilin90) [inline tidb-cloud/release-notes-2022.md:575]:
```suggestion
* Update the [billing information](/tidb-cloud/tidb-cloud-billing.md) and the [price table](https://www.pingcap.com/pricing/).
```

File: tidb-cloud/tidb-cloud-billing-dm.md [hunk 7]
```diff
@@ -51,7 +51,7 @@ Note that if you are using AWS PrivateLink or VPC peering connections, and if th
 
     ![Cross-region and cross-AZ traffic charges](/media/tidb-cloud/dm-billing-cross-region-and-az-fees.png)
 
-The cross-region and cross-AZ traffic prices are the same as those for TiDB Cloud. For more information, see [TiDB Cloud Pricing Details](https://en.pingcap.com/tidb-cloud-pricing-details/).
+The cross-region and cross-AZ traffic prices are the same as those for TiDB Cloud. For more information, see [TiDB Cloud Pricing Details](https://www.pingcap.com/tidb-cloud-pricing-details/).
```
Comment (lilin90) [inline tidb-cloud/tidb-cloud-billing-dm.md:54]:
```suggestion
The cross-region and cross-AZ traffic prices are the same as those for TiDB Cloud. For more information, see [TiDB Cloud Pricing Details](https://www.pingcap.com/tidb-dedicated-pricing-details/).
```

File: tidb-cloud/tidb-cloud-billing.md [hunk 8]
```diff
@@ -12,7 +12,7 @@ summary: Learn about TiDB Cloud billing.
 TiDB Cloud charges according to the resources that you consume. You can visit the following pages to get more information about the pricing.
 
 - [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details)
-- [TiDB Dedicated Pricing Details](https://en.pingcap.com/tidb-cloud-pricing-details/)
+- [TiDB Dedicated Pricing Details](https://www.pingcap.com/tidb-cloud-pricing-details/)
```
Comment (lilin90) [inline tidb-cloud/tidb-cloud-billing.md:15]:
```suggestion
- [TiDB Dedicated Pricing Details](https://www.pingcap.com/tidb-dedicated-pricing-details/)
```

File: tidb-cloud/tidb-cloud-billing.md [hunk 9]
```diff
@@ -12,7 +12,7 @@ summary: Learn about TiDB Cloud billing.
 TiDB Cloud charges according to the resources that you consume. You can visit the following pages to get more information about the pricing.
 
 - [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details)
```
Comment (lilin90) [inline tidb-cloud/tidb-cloud-billing.md:14]:
```suggestion
- [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-serverless-pricing-details/)
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**